<launch>

    <!-- 将 Urdf 文件的内容加载到参数服务器 -->
    <env name="GAZEBO_PLUGIN_PATH" value="$(env HOME)/.gazebo/plugins"/>
    <!-- 启动 gazebo -->
    <arg name="world_name" default="trees" />
    <include file="$(find gazebo_ros)/launch/empty_world.launch">
        <arg name="world_name" value="$(find sdpc)/worlds/$(arg world_name)" />
        <arg name="paused" value="false" />
        <arg name="use_sim_time" value="true" />
        <arg name="gui" value="true" />
        <arg name="headless" value="false" />
        <arg name="debug" value="false" />
        <arg name="verbose" value="true" />
    </include>
</launch>
